include/mav/utils.h

```diff
@@ -100,13 +100,6 @@ namespace mav {
     }
 
 
-    inline uint64_t millis() noexcept {
-        return std::chrono::duration_cast<std::chrono::milliseconds>(
-                std::chrono::system_clock::now().time_since_epoch()).count();
-    }
```

Comment:
I would hope it's safe to remove this public helper function.

If we're worried, we could keep and at least make it return a steady clock time.

Reply:
I don't know how far reaching is this, within this project it was only used only once. We can deprecate the method first, so it would print a compile-time warning with a clear message and get deleted with next version bump, e.g.
```
    [[deprecated("millis() method uses non-monotonic clock source and it will be removed in the next release.")]] inline uint64_t millis() noexcept {
        return std::chrono::duration_cast<std::chrono::milliseconds>(
                std::chrono::system_clock::now().time_since_epoch()).count();
    }
``` 

I would strongly suggest to remove it, since it's up to the developer to select an appropriate clock source for the task.